src/app/shared/form/fancy-duration.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, Input, DoCheck } from '@angular/core';
+import { BaseModel } from '../model/base.model';
+
+@Component({
+  selector: 'publish-fancy-duration',
+  styleUrls: ['fancy-field.component.css', 'fancy-duration.component.css'],
+  template: `
+    <div class="field small inline" [class.tiny]="tiny">
+      <h4 *ngIf="label">
+        <label [attr.for]="hoursName">{{label}}</label>
+      </h4>
+      <template [ngIf]="!model">
+        <input [id]="hoursName" type="number" disabled=true/>
```

Comment:
Yeah, the up/down arrows are real handy, if you're tabbing through these fields.